pick.lic

```diff
@@ -231,6 +232,11 @@ class Pick
     @equipment_manager.wear_items(@removed_items)
   end
 
+  def balance_lockpick_container
+    return unless @use_lockpick_ring and @balance_lockpick_container
```

Comment:
I think you'll need something at the top to pull @settings.balance_lockpick_container into the class variable @balance_lockpick_container, and probably to add it to the debug output of all the settings.